website/docs/configuration/modules.html.md

```diff
@@ -416,9 +419,8 @@ after performing such actions to ensure that no resources are surprisingly
 deleted.
 
 Each instance of a module may optionally have different providers passed to it
-using the `providers` argument described above. This can be useful in situations
-where, for example, a duplicated set of resources must be created across
-several regions or datacenters.
+using the [`providers`](/docs/configuration/modules.html#passing-providers-explicitly)
```

Comment:
I think this statement isn't accurate when now applied to this new meaning of "multiple instances", because the `providers` argument is static and therefore must be identical for all instances created from the same `module` block. Does that seem right, or did I misunderstand your intent here?

Reply:
yes, I think I misunderstood here, that was part of trying to merge the module expansion and this information, without getting into the limitations as yet. Maybe I'll get that section started and then expand it later 🤔

Reply:
Mind checking the update to ensure what I've got is correct? (mostly focusing on correctness, and then we can circle back later to improve)

Reply:
Yes, this matches my understanding of the behavior now. :+1: